ext/cl_khr_external_memory.asciidoc

```diff
@@ -586,3 +586,11 @@ while (true) {
 }
 ----
 --
+
+
+=== Issues
+
+. How should the import of images that are created in external APIs with non-linear tiling be robustly handled?
++
+--
+*UNRESOLVED*
```

Comment:
```suggestion
*UNRESOLVED*
--
```

Can you please add the closing block to this issue?  Without this fix I'm getting the error:

> asciidoctor: WARNING: OpenCL_Ext.txt: line 111: unterminated open block